frontend/packages/ceph-storage-plugin/src/components/independent-mode/install.tsx

```diff
@@ -76,42 +51,33 @@ const InstallExternalCluster = withHandlePromise((props: InstallExternalClusterP
     const file = event.target.files[0];
     reader.onload = (ev) => {
       const data = ev.target?.result as string;
+      if (isValidJSON(data)) {
+        setDataError(checkError(data, plainKeys, encodedKeys));
+      } else {
+        setDataError('The uploaded file is not a valid JSON file');
+      }
```

Comment:
```suggestion
      isValidJSON(data) ? setDataError(checkError(data, plainKeys, encodedKeys)) : setDataError(checkError(data, plainKeys, encodedKeys));
```

Reply:
This would be nice. But I am adding another statement inside the `if` block.